src/sections/file/file-embargo/FileEmbargoDate.tsx

```diff
@@ -12,23 +12,29 @@ interface FileEmbargoDateProps {
 export function FileEmbargoDate({
   embargo,
   datasetPublishingStatus,
-  format = 'short'
+  format = 'YYYY-MM-DD'
 }: FileEmbargoDateProps) {
   const { t } = useTranslation('files')
 
   if (!embargo) {
     return <></>
   }
 
-  // TODO: use time tag with dateTime attr https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
-
   return (
     <div>
       <span>
         {t(embargoTypeOfDate(embargo.isActive, datasetPublishingStatus))}{' '}
-        {format === 'YYYY-MM-DD'
-          ? DateHelper.toDisplayFormatYYYYMMDD(embargo.dateAvailable)
-          : DateHelper.toDisplayFormat(embargo.dateAvailable)}
+        <time
+          dateTime={
+            format === 'YYYY-MM-DD'
```

Comment:
datetime value is the machine-readable value of the datetime, so in there you shouldn't use the `displayFormat` which is a a more human readable but the `toISO8601Format` one.